### PR TITLE
Fix filter bug and revert to sentry CDN

### PR DESCRIPTION
**Pull request by ctaylor451**

### CODE REVIEW

File: main/templates/base_generic.html [hunk 1]
```diff
@@ -69,10 +69,14 @@
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
     {% endblock js %}
     {% comment %}
-      browser.sentry-cdn.com is blocked by ad blockers so this copy needs to be
+    moved back to cdn. Do we still see this problem?
+    [obsolete?] browser.sentry-cdn.com is blocked by ad blockers so this copy needs to be
```
Comment (kduncklee) [inline main/templates/base_generic.html:73]: Why do you think that ad blockers will have changed? Is it not better to keep self-hosting this?

**Comment by ctaylor451** [comment]
My ad blocker didn't see it.  My concern is we are not updating it

Do you remember the test case last time?

**Comment by kduncklee** [comment]
It sounds like it occasionally gets added to various blocking lists. I think last time it was EasyPrivacy.
https://docs.sentry.io/platforms/javascript/#verifying-your-setup has some more details.

With the above implementation, we still need to manually update the version number (5.20.1).

**Comment by ctaylor451** [comment]
OK, I'll revert that part of the request

**Pull request merged**